esp-hal/src/i2c.rs

```diff
@@ -1528,6 +1545,7 @@ pub trait Instance: crate::private::Sealed {
     }
 
     #[cfg(not(any(esp32, esp32s2)))]
+    /// Reads all bytes from the RX FIFO
```

Comment:
I'm not sure I want to be as pedantic as the LLVM people, but most other comments end with a period, and I guess we should be consistent in that :)

```suggestion
    /// Reads all bytes from the RX FIFO.
```